src/App.js

```diff
@@ -1,25 +1,64 @@
 import React from 'react';
 
 import './App.scss';
+import { Clock } from './components/Clock';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: Math.trunc(Math.random() * 1000),
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  showClock = () => {
+    this.setState({
+      isClockVisible: true,
+    });
+  }
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+  hideClock = () => {
+    this.setState({
+      isClockVisible: false,
+    });
+  }
+
+  randomName = () => {
```

Comment:
это функция и должно называться как указание действия

установиРандомноеИмя
```suggestion
  setRandomName = () => {
```